package-lock.json

```diff
@@ -1,27 +1,6 @@
 {
+  "name": "z3",
```

Comment:
this file should not be in the pull request

Reply:
It seems @NikolajBjorner has added it 
C:\dev\z3>git log .\package-lock.json
commit b3f5cbe3f4f1727b8df1a1e1cbc9438f2f85466f
Author: Nikolaj Bjorner <nbjorner@microsoft.com>
Date:   Thu Aug 17 02:09:27 2023 +0000

    add parameters

    Signed-off-by: Nikolaj Bjorner <nbjorner@microsoft.com>

Reply:
Should I restore it to the previous version?